### PR TITLE
Prime1: Bulk Database Video Upload

### PR DESCRIPTION
**Pull request by Samuel6710**
52 vids added from the Prime Database Video Tracking Sheet doc (https://docs.google.com/spreadsheets/d/11v0YjiuxwJUB1DmJ9990DlF_uLqaYsPxYh5Bl6IXp7g/edit?usp=sharing)

### CODE REVIEW

File: randovania/games/prime1/json_data/Phazon Mines.txt [hunk 1]
```diff
@@ -1474,7 +1493,9 @@ Extra - aabb: [-108.57961, -160.31125, -168.22823, 77.58658, -50.59403, -92.4295
           Any of the following:
               Space Jump Boots or Slope Jump (Beginner)
```
Comment (JeffGainsNGames) [inline randovania/games/prime1/json_data/Phazon Mines.txt:1494]: slope jump beginner was a part of the NSJ method video
https://youtu.be/MwycHUSwAaQ?t=9

File: randovania/games/prime1/json_data/Phazon Mines.txt [hunk 2]
```diff
@@ -1474,7 +1493,9 @@ Extra - aabb: [-108.57961, -160.31125, -168.22823, 77.58658, -50.59403, -92.4295
...
               Morph Ball Bomb and Morph Ball and Bomb Jump (Beginner)
-              Scan Visor and Combat/Scan Dash (Beginner) and Standable Terrain (Beginner)
+              All of the following:
+                  # https://youtu.be/MwycHUSwAaQ
```
Comment (JeffGainsNGames) [inline randovania/games/prime1/json_data/Phazon Mines.txt:1497]: the nsj slope jump (beginner) method above was also captured in this video: https://youtu.be/MwycHUSwAaQ?t=9

File: CHANGELOG.md [hunk 3]
```diff
@@ -64,6 +64,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Logic Database
 
+- Added: 52 videos to logic database
```
Comment (JeffGainsNGames) [inline CHANGELOG.md:67]: replace with: 
- Added: 52 videos to logic database, bringing the total available via the [Video Directory](https://randovania.github.io/Metroid%20Prime/) to 276

see previous changelog updates as example

**Review by JeffGainsNGames — CHANGES_REQUESTED** [review]

**Review by JeffGainsNGames — APPROVED** [review]

File: randovania/games/prime1/json_data/Tallon Overworld.txt [hunk 4]
```diff
@@ -1329,7 +1347,9 @@ Extra - aabb: [-110.4026, 121.974915, -95.92104, -23.443779, 152.64822, -38.8491
           Morph Ball
           Any of the following:
               Morph Ball Bomb and Gravity Suit
-              Boost Ball and Wall Boost (Advanced)
+              All of the following:
+                  # https://youtu.be/3s1SrxoMF_Y or https://youtu.be/ih5KCjnUE_0 (With Gravity Suit)
```
Comment (toasterparty) [inline randovania/games/prime1/json_data/Tallon Overworld.txt:1351]: The database video extractor can't handle two videos in the same comment

File: randovania/games/prime1/json_data/Tallon Overworld.txt [hunk 5]
```diff
@@ -1684,6 +1706,7 @@ Extra - aabb: [-54.134953, -94.6864, 2.2145255, 68.01861, 53.576847, 101.807175]
               All of the following:
                   Boost Ball or Spinners without Boost (Advanced)
                   Any of the following:
+                      # https://youtu.be/F2K62ZJ0lxM or https://www.youtube.com/watch?v=Kn65gJzeOwA (Manip with Gravity Suit)
```
Comment (toasterparty) [inline randovania/games/prime1/json_data/Tallon Overworld.txt:1709]: The database video extractor can't handle two videos in the same comment

File: randovania/games/prime1/json_data/Tallon Overworld.txt [hunk 6]
```diff
@@ -1312,7 +1328,9 @@ Extra - aabb: [-110.4026, 121.974915, -95.92104, -23.443779, 152.64822, -38.8491
           Morph Ball
           Any of the following:
               Morph Ball Bomb and Gravity Suit
-              Boost Ball and Wall Boost (Advanced)
+              All of the following:
+                  # https://youtu.be/3s1SrxoMF_Y or https://youtu.be/ih5KCjnUE_0 (With Gravity Suit)
```
Comment (toasterparty) [inline randovania/games/prime1/json_data/Tallon Overworld.txt:1332]: The database video extractor can't handle two videos in the same comment

File: randovania/games/prime1/json_data/Tallon Overworld.txt [hunk 7]
```diff
@@ -1298,7 +1312,9 @@ Extra - aabb: [-110.4026, 121.974915, -95.92104, -23.443779, 152.64822, -38.8491
...
```
Comment (toasterparty) [inline randovania/games/prime1/json_data/Tallon Overworld.txt:1316]: The database video extractor can't handle two videos in the same comment

**Review by toasterparty — CHANGES_REQUESTED** [review]

**Review by toasterparty — APPROVED** [review]

**Pull request merged**